README.md

```diff
@@ -1,15 +1,60 @@
-[![TLA+ Build Status](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
+Overview
+--------
+[![Build CI](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
```

Comment:
Use fewer acronyms (CI, CLI, ...) or introduce them first.

Reply:
In this case the text here is actually immaterial, it just uses the name of the workflow in the badge which is just CI

Reply:
Added definitions of CLI and IDE